app/workers/evss/retrieve_claims_from_remote_job.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'evss/common_service'
+
+module EVSS
+  class RetrieveClaimsFromRemoteJob
+    include Sidekiq::Worker
+    sidekiq_options retry: false
+
+    def perform(user_uuid)
+      Sentry::TagRainbows.tag
+      @user = User.find user_uuid
+      tracker = EVSSClaimsSyncStatusTracker.new(user_uuid: user_uuid)
+      auth_headers = EVSS::AuthHeaders.new(@user).to_h
+      @client = EVSS::ClaimsService.new(auth_headers)
+      raw_claims = @client.all_claims.body
+      EVSSClaimService::EVSS_CLAIM_KEYS.each_with_object([]) do |key, claim_accum|
+        next unless raw_claims[key]
+        claim_accum << raw_claims[key].map do |raw_claim|
+          create_or_update_claim(raw_claim)
+        end
+      end
+      tracker.set_collection_status('SUCCESS')
+    rescue StandardError
```

Comment:
Since we're not allowing retries, I removed the `sidekiq_retries_exhausted` and added this `rescue` in order to set the status to 'FAILED'